### ISSUE TITLE
karabo.test.test_pinocchio.TestPinocchio fails on GH

### ISSUE THREAD

**Issue by deiruch** [i4Ds/Karabo-Pipeline#126]
See https://github.com/i4Ds/Karabo-Pipeline/runs/7915455867

```
Traceback (most recent call last):
  File "/__w/Karabo-Pipeline/Karabo-Pipeline/karabo/test/test_pinocchio.py", line 37, in testSimpleInstance
    sky = p.getSkyModel()
  File "/__w/Karabo-Pipeline/Karabo-Pipeline/karabo/simulation/pinocchio.py", line 622, in getSkyModel
    return Pinocchio.getSkyModelFromFiles(self.outLightConePath, near, far)
  File "/__w/Karabo-Pipeline/Karabo-Pipeline/karabo/simulation/pinocchio.py", line 652, in getSkyModelFromFiles
    (x,y,z)=np.loadtxt(path, unpack=True, usecols=(2,3,4))
  File "/opt/conda/envs/karabo_dev_env/lib/python3.9/site-packages/numpy/lib/npyio.py", line 1042, in loadtxt
    fh = np.lib._datasource.open(fname, 'rt', encoding=encoding)
  File "/opt/conda/envs/karabo_dev_env/lib/python3.9/site-packages/numpy/lib/_datasource.py", line 193, in open
    return ds.open(path, mode, encoding=encoding, newline=newline)
  File "/opt/conda/envs/karabo_dev_env/lib/python3.9/site-packages/numpy/lib/_datasource.py", line 532, in open
    raise FileNotFoundError(f"{path} not found.")
FileNotFoundError: /__w/Karabo-Pipeline/Karabo-Pipeline/.tmp/2afdb5b2-7ddc-45ef-a1a4-059cced4cb77/pinocchio.unittest.plc.out not found.
```

**Comment by fschramka** [comment]
seems like pinocchio is not running - mpirunner seems to have a problem on the action server

**Comment by fschramka** [comment]
Fixed in [7382354]
